common/libimage/load.go

```diff
@@ -110,6 +111,11 @@ func (r *Runtime) Load(ctx context.Context, path string, options *LoadOptions) (
 			return loadedImages, err
 		}
 		logrus.Debugf("Error loading %s (%s): %v", path, transportName, err)
+
+		if errors.Is(err, syscall.ENOSPC){
+            return nil, fmt.Errorf("no space left on device")
```

Comment:
please do do that, that throws away the full error context, i.e. you don't even know which file was attempted to write.

This must still return err.